src/analysis/excluded_terms_in_mondo/excluded_terms_in_mondo.py

```diff
@@ -0,0 +1,167 @@
+"""Excluded terms in / xrefed-in Mondo
+
+Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/31
+
+TODO's
+"""
+import os
+import sys
+from argparse import ArgumentParser
+from typing import Dict, List
+
+import pandas as pd
+
+
+# Vars
+# # Static
+THIS_SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+# CACHE_DIR = os.path.join(THIS_SCRIPT_DIR, 'cache')
+ANALYSIS_DIR = os.path.join(THIS_SCRIPT_DIR, '..')
+PROJECT_DIR = os.path.join(ANALYSIS_DIR, '..', '..')
+MAPPINGS_DIR = os.path.join(THIS_SCRIPT_DIR, 'ignored', 'mappings')
+SIGNATURE_FILES_DIR = os.path.join(PROJECT_DIR, 'src', 'ontology', 'reports')
+# # Config
+CONFIG = {
+    'ontologies': ['doid', 'icd10cm', 'icd10who', 'ncit', 'omim', 'ordo'],  # todo: get list from mondo config yml
```

Comment:
I think there's somewhere, like a mondo config yml, where I can get this list of names, but I don't remember where at the moment. I saved the note somewhere tho.

Reply:
redundant with same comment below